docs/samples/git-setup/clustertemplates/version_4.Y.Z/sno-ran-du/sno-ran-du-v4-Y-Z-6-full-DU.yaml

```diff
@@ -0,0 +1,313 @@
+apiVersion: o2ims.provisioning.oran.org/v1alpha1
+kind: ClusterTemplate
+metadata:
+  name: sno-ran-du.v4-Y-Z-6
+  namespace: sno-ran-du-v4-Y-Z
+spec:
+  name: sno-ran-du
+  version: v4-Y-Z-6
+  templates:
+    hwTemplate: placeholder-du-template-configmap-v1
+    clusterInstanceDefaults: clusterinstance-defaults-v5
+    policyTemplateDefaults: policytemplate-defaults-v3
+  templateParameterSchema:
+    properties:
+      nodeClusterName:
+        type: string
+      oCloudSiteId:
+        type: string
+      policyTemplateParameters:
+        description: policyTemplateSchema defines the available parameters for cluster configuration
+        properties:
+          cluster-log-fwd-filters:
+            type: string
+          cluster-log-fwd-outputs:
+            type: string
+          cluster-log-fwd-pipelines:
```

Comment:
It has been agreed that this will be locked in the new version of the ClusterLogForwarder, currently WIP under OCPBUGS-44518, so it will be removed from this ClusterTemplate.